### DISCUSSION TITLE
Add support for short links

### DISCUSSION THREAD

**Discussion by windigerbus** [shaked6540/YoutubePlaylistDownloader#164] [Ideas]
When you copy the link for shorts it comes out as "youtube.com/shorts/whatever", which you can't download for some reason, but it's also accessible through "youtube.com/watch?v=whatever". It's pretty annoying to change the link everytime to download shorts, so you could maybe automatically convert them.

**Comment by shaked6540** [comment]
Great suggestion, I'll work on it when I have time

**Comment by shaked6540** [comment]
Done in 1.9.19, make sure to update the program

**Comment by windigerbus** [comment]
Thanks, this will make my life so much easier :)
